blocks/card-upload-document/src/CardUploadDocument.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useCallback } from "react";
+import { useDropzone } from "react-dropzone";
+import { Typography } from "@mui/joy";
+import Box from "@mui/joy/Box";
+import Button from "@mui/joy/Button";
+import Card from "@mui/joy/Card";
+import CardContent from "@mui/joy/CardContent";
+import SvgIcon from "@mui/joy/SvgIcon";
+
+export default function CardUploadDocument() {
+  const onDrop = useCallback((acceptedFiles: File[]) => {
+    // Do something with the files
+    console.log(acceptedFiles);
+  }, []);
+  const { getRootProps, isDragActive } = useDropzone({
+    onDrop,
+    noClick: true,
+    noKeyboard: true,
+  });
+
+  return (
+    <div {...getRootProps()}>
+      <Card
```

Comment:
```suggestion
      <Card
        {...getRootProps()}
```
It's better to spread the props directly to the card

Reply:
I encountered a TypeScript error while attempting to spread the props into the Card component."

<img width="631" alt="image" src="https://github.com/siriwatknp/joy-treasury/assets/22396258/39871756-eac7-4529-90c7-26f41e29f36b">

Reply:
Found a way to make TypeScript happy in https://github.com/siriwatknp/joy-treasury/pull/95/commits/eccf23bb6c5a5035d9a58819e881666b0e073d80:

```ts
<Card
  {...(getRootProps() as Omit<DropzoneRootProps, "color">)}
  variant="plain"
  sx={{ minWidth: 300, borderRadius: "xl", boxShadow: "lg" }}
  size="sm"
>
```